### PR TITLE
data:tanitoluwam

### PR DESCRIPTION
**Pull request by adenugatani**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5073"></a>

### CODE REVIEW

**Review by SaraJaoude — APPROVED** [review]
looks good

File: tanitoluwam.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "name": "Instagram",
+      "url": "https://instagram.com/tanii_toluwa",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/adenugatani",
+      "icon": "FaLinkedIn"
```
Comment (swagataroy30) [inline tanitoluwam.json:24]: There is a typo in the icon name at line 24 it should be `FaLinkedin`

File: tanitoluwam.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+     {
+      "name": "GitHub",
+      "url": "https://linkedin.com/tanitoluwam",
```
Comment (swagataroy30) [inline tanitoluwam.json:28]: You might want to add your Github profile url here at line 28

File: tanitoluwam.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "icon": "FaGitHub"
```
Comment (swagataroy30) [inline tanitoluwam.json:29]: There is a typo in the icon name at line 29 it should be `FaGithub`

**Comment by tanitoluwam** [comment]
> The `json` file is not in the `data` folder. It should be moved to the `data` folder.

i dont understand how you mean, my file is in the data folder

**Comment by eddiejaoude** [comment]
@swagataroy30 thank you for noticing 👍 I will move to the data folder now

@tanitoluwam your file is in the root of the project not in the `data` folder

**Pull request merged**